src/core/store/columnContext.tsx

```diff
@@ -38,15 +39,23 @@ class PanelsRepository {
     panel.resizableNode.style.height = newHeight + 'px';
   }
 
-  /* Returns array with [panel, panelHeight] entries */
-  getPanelsWithExtraSpace() {
-    return Array.from(this.panels).reduce((acc: [PanelMeta, number][], c) => {
-      const height = c.resizableNode.getBoundingClientRect().height;
-      if (height > c.minHeight) {
-        acc.push([c, height]);
-      }
-      return acc;
-    }, []);
+  getPanelsWithExtraSpace(): [[PanelMeta, number, number][], number] {
+    let totalExtraSpace = 0;
+    /* Returns array with [panel, panelHeight, availableExtraSpace ] entries */
```

Comment:
This comment not aligned with `[[PanelMeta, number, number][], number]`

Reply:
I think it would be cleaner if you choose one approach:
forEach + variables outside from cycle for accumulate values, or reduce. Currently it's mix of those without visible reason

Reply:
Okay, but is it wrong to increment outside variable from reduce callback?

Reply:
the point of reduce is to give you a function that already has this variable in scope